test-case/check-playback.sh

```diff
@@ -97,6 +97,15 @@ do
                     func_lib_lsof_error_dump "$snd"
                     die "aplay on PCM $dev failed at $i/$loop_cnt."
                 }
+
+		tplg_basename=$(basename $tplg)
+		platform=$(sof-dump-status.py -p)
+		is_ipc4_zephyr && {
+		    data_file=$LOG_ROOT/mtrace.txt
```

Comment:
AFAIK this new test is merely scanning logs so it should be hardcoded to neither IPC4 nor Zephyr nor mtrace.

Reply:
@marc-hb  Well that's complicated. The log output structure is specific to Zephyr (it uses formatting of the Zephyr logging subsystem). IPC4 should not be a dependency, but we do need separate code to handle sof-logger output.

Reply:
One big TODO that needs to be addresses is limiting the analysis to also cover logs that happened during the test run. Here full mtrace.txt is analyzed, but it is not guaranteed the DSP went to D3 between the test iterations. So some kind of capture of the timestamp of last line of log line before test starts, and pruning the log to only cover the new entries added during test execution, is needed.

Reply:
Seems complicated...  
1) Can the analysis be ran last, after the log contributors have finished?
2) Can the analysis be ran as a "follow," so it gathers updates automatically?
3) Can the logs be backed up and purged after the initial run?  

It's possible 3 may not work, as it may still present the same challenges.